### ISSUE TITLE
Datetime mixup in recurrences

### ISSUE THREAD

**Issue by BastianBalthasarBux** [daCyberpunk/dated_news#14]
I noticed that the recurrences are always 2hrs after the first given start/stop.
I suspect this happens because (while correct server+typo3 settings) of PHP DateTime to Europe/Vienna, and we have summertime. Somewhere the 2hrs+ to UTC get appended although they shouldn't.
Any hints, where this could be?

**Comment by BastianBalthasarBux** [comment]
Yeah, it must be timezone related: if I add a recurrence which is recurred in summertime AND wintertime, I lose an hour and it's only +1hrs ... somewhere the php timezone settings are mixed, or something like that ...

**Comment by bocianl** [comment]
The same here. One event has correct time other +1 hour :(
